resource_calendar_leaves_province/models/resource_calendar_leaves.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import fields, models
+
+
+class ResourceCalendarLeaves(models.Model):
+    _inherit = "resource.calendar.leaves"
+
+    state_ids = fields.Many2many(
+        comodel_name="res.country.state",
+        string="States",
+        help="Provinces for which this leave is applicable",
```

Comment:
issue: I think the word "state" is hard enough to translate. If we change it, we make it even harder (for non-spaniards).

```suggestion
        help="States for which this leave is applicable",
```